isis/scripts/install_isis.sh

```diff
@@ -0,0 +1,472 @@
+            ;;
+    esac
+done
+
+set -- "${POSITIONAL_ARGS[@]}" # restore positional parameters
+
+# If an ANACONDA_LABEL was not set, ask the user for it
+if [[ -z $ANACONDA_LABEL ]]; then
+    ANACONDA_LABEL="main"
+    printf "ISIS anaconda label [$ANACONDA_LABEL] will be installed.\n"
+    printf "\\n"
+    printf "  - Press ENTER to confirm the ISIS label to install\\n"
+    printf "  - Press CTRL-C to abort the installation\\n"
+    printf "  - Or specify a different ISIS label below [main, dev, lts, rc]\\n"
+    printf "  - Default is [main]\\n"
+    printf "\\n"
+    printf "[%s] >>> " "$ANACONDA_LABEL"
+
+    read -r anaconda_label
+
+    # If a label was given, set it
+    if [ -n "$anaconda_label" ]; then
+        ANACONDA_LABEL=$anaconda_label
+    fi
+fi
+
+# Force label to uppercase for LTS and RC
+if [ "${ANACONDA_LABEL,,}" = "lts" ] || [ "${ANACONDA_LABEL,,}" = "rc" ]; then
+    ANACONDA_LABEL="${ANACONDA_LABEL^^}"
+fi
+
+printf "\nISIS anaconda label set to [$ANACONDA_LABEL]\n"
+
+if [ "$ANACONDA_LABEL" = "main" ]; then
+    PACKAGE_NAME="usgs-astrogeology::isis"
+else
+    PACKAGE_NAME="usgs-astrogeology/label/$ANACONDA_LABEL::isis"
+fi
+
+# If an ISIS_VERSION was not set, ask the user for it
+if [[ -z $ISIS_VERSION ]]; then
+    ISIS_VERSION="latest"
+    # Skip the version prompt if ANACONDA_LABEL is dev
+    if [ "$ANACONDA_LABEL" != "dev" ]; then
+        printf "Specify an ISIS version to install from the ISIS anaconda label [$ANACONDA_LABEL]\n"
+        printf "\\n"
+        printf "  - Press ENTER to install the latest version of ISIS found under [$ANACONDA_LABEL]\\n"
+        printf "  - Press CTRL-C to abort the installation\\n"
+        printf "  - Or specify an ISIS version below\\n"
+        printf "\\n"
+        printf "[%s] >>> " "$ISIS_VERSION"
+
+        read -r isis_version
+
+        # If a label was given, set it
+        if [ -n "$isis_version" ]; then
+            ISIS_VERSION=$isis_version
+        fi
+        printf "\nISIS version set to [$ISIS_VERSION]\n"
+    else
+        printf "\nUsing latest version from dev channel\n"
+        ISIS_VERSION="latest"
+    fi
+else
+    printf "\nISIS version set to [$ISIS_VERSION]\n"
+fi
+
+if [ $ISIS_VERSION = "latest" ]; then
+    ISIS_VERSION=""
+fi
+
+printf "\nBeginning install of $PACKAGE_NAME\n\n"
+
+# Determine the OS type
+case "$(uname)" in
+    "Linux")
+        MINIFORGE_INSTALLER="Miniforge3-Linux-x86_64.sh"
+        LIBGL_INSTALL=""
+        ;;
+    "Darwin")
+        MINIFORGE_INSTALLER="Miniforge3-MacOSX-x86_64.sh"
+        LIBGL_INSTALL=""
+        export CONDA_SUBDIR=osx-64
+        ;;
+    *)
+        echo "Unsupported OS: $(uname)"
+        exit 1
+        ;;
+esac
+
+# Check if conda is installed but mamba is not
+if command -v conda &> /dev/null && ! command -v mamba &> /dev/null; then
+    echo "WARNING: conda is installed but mamba is not."
+    echo "mamba is a faster alternative to conda and is recommended for ISIS installation."
```

Comment:
The word "recommended" seams to weak here. If the user passes "no" in response to the prompt, the `if` block on Line 221 prints a message that only mamba is supported and the script exits. 

The prompt should be clear that mamba is actually _required_ or the script should be modified to proceed with an existing install of conda without mamba.